apps/web/src/server/applications/create/applicant/applications-create-applicant.controller.js

```diff
@@ -68,7 +67,15 @@ export async function updateApplicationsCreateApplicantTypes({ path, session, bo
  * @type {import('@pins/express').RenderHandler<ApplicationsCreateApplicantOrganisationNameProps, {}, {}, {}, {}>}
  */
 export async function viewApplicationsCreateApplicantOrganisationName(req, response) {
-	response.render('applications/create/applicant/_organisation-name');
+	const { applicationId } = response.locals;
+
+	const { applicants } = await getApplicationDraft(applicationId);
+	const applicant = applicants && applicants.length > 0 ? applicants[0] : null;
```

Comment:
you use the code in line 72/73 in every function, makes sense to abstract it into a reusable function